### DISCUSSION TITLE
Video titles not fully showing on my 13 inch screen.

### DISCUSSION THREAD

**Discussion by bayegaspard** [OpenMined/courses#41] [Feedback] [answered]
Does someone with 13 inch screen also experience the problem with titles on videos not fully showing?

**Comment by yemikifouly** [comment, marked as answer]
Hum... Would you mind sharing a screenshot of what that looks like please? Thanks! 🙂

**Reply by bayegaspard** [reply]
Hi Yemi, Issue fixed thank you.I can now fully see it.

**Reply by yemikifouly** [reply]
Oh okay! It wasn't a bug with the website though, was it?! Because if it was I would like to file it for our Web Developer.

**Reply by bayegaspard** [reply]
Apparently it wasn't a bug on the platform.If I find any bug I  will let you know.)) Thank you.

**Reply by yemikifouly** [reply]
Awesome! 🙂 Thank you too!
